modules/govuk/manifests/node/s_api_elasticsearch.pp

```diff
@@ -65,25 +65,6 @@
       from    => getparam(Govuk_host['calculators-frontend-3'], 'ip'),
       require => Govuk_host['calculators-frontend-3'],
     }
-
-    # FIXME: Remove these firewall rules by moving the need-api app from the backend
-    # machines to the API machines. The MongoDB database will need to move too.
-    # There are more firewall rules in govuk-provisioning that can also be removed.
-    @ufw::allow { 'allow-elasticsearch-http-9200-from-backend-1':
```

Comment:
Should this be `ensure: 'absent'`-ed in the previous PR before we remove it outright in this one?

Reply:
I've added `ensure: absent` in the previous PR https://github.com/alphagov/govuk-puppet/pull/6425/commits/d0eaf7b7e633f0cce572344d7a494ddc43f4b8c4#diff-2f7f03a377299e06b51f7d1b66106db8L72

Reply:
Doesn't look like that works so I will un-do it https://ci.integration.publishing.service.gov.uk/job/govuk-puppet/job/remove-need-api/16/console